pkgs/applications/version-management/git-together/default.nix

```
@@ -0,0 +1,31 @@
+{ pkgs, lib, stdenv, fetchFromGitHub, rustPlatform, openssl, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "git-together";
+  version = "v0.1.0-alpha.26";
+
+  src = fetchFromGitHub {
+    owner = "kejadlen";
+    repo = pname;
+    rev = version;
+    hash = "sha256-2HgOaqlX0mmmvRlALHm90NAdIhby/jWUJO63bQFqc+4=";
+  };
+
+  buildInputs = lib.optional stdenv.isDarwin Security;
+
+  preBuild = ''
+    export OPENSSL_NO_VENDOR=1
+    export OPENSSL_DIR="${openssl.dev}"
+    export OPENSSL_LIB_DIR="${openssl.out}/lib"
+  '';
```

```suggestion
  nativeBuildInputs = [pkg-config];

  buildInputs = [openssl] ++ lib.optional stdenv.isDarwin Security;

  OPENSSL_NO_VENDOR = true;

  OPENSSL_LIB_DIR=lib.mkIf stdenv.isDarwin (lib.getLib openssl);
```
Don't forget to also add pkg-config to the top

---

That did the trick! The `OPENSSL_LIB_DIR` line didn't work, but it build ok without out on mac & linux. I added the `OPENSSL_NO_VENDOR` late in my attempts at getting libgit to look in the right place.

---

Oh, nice!